src/libraries/System.Net.Security/src/System/Net/Security/SslStream.Implementation.cs

```diff
@@ -396,169 +401,6 @@ private bool CompleteHandshake(ref ProtocolToken? alertToken)
             return true;
         }
 
-        private void FinishHandshakeRead(int newState)
```

Comment:
I love that all of this gets deleted.

One step closer to removing LazyAsyncResult entirely :)